app/controllers/catalog_controller.rb

```diff
@@ -72,7 +72,7 @@ class CatalogController < ApplicationController
     # exploded_tag_ssim indexes all tag prefixes (see IdentityMetadataDS#to_solr for a more exact
     # description), whereas tag_ssim only indexes whole tags.  we want to facet on exploded_tag_ssim
     # to get the hierarchy.
-    config.add_facet_field 'exploded_tag_ssim',               label: 'Tag',                 partial: 'blacklight/hierarchy/facet_hierarchy'
+    config.add_facet_field 'exploded_tag_ssim',               label: 'Tag',                 limit: 9999, partial: 'blacklight/hierarchy/facet_hierarchy'
```

Comment:
> facet.limit
>
> This param indicates the maximum number of constraint counts that should be returned for the facet fields. A negative value means unlimited.

Should this be -1, instead of simply a large number?